dmoj/cptbox/ptproc.cpp

```diff
@@ -105,6 +105,15 @@ bool pt_process::use_seccomp(bool enabled) {
     return true;
 }
 
+bool pt_process::use_landlock(bool enabled) {
```

Comment:
Is the return value of this function supposed to be if turning landlock on was successful? If so, it should return `false` if we couldn't turn it on because we don't have landlock support.